src/pages/Archive/Archive.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { useNotesContext } from "../../utils/contexts/NotesContext";
+import Note from "../../components/Note";
+function Notes() {
+  const { notes, saveNotes } = useNotesContext();
+
+  const renderNotes = () => {
+    return notes
+      .filter((f) => f.archived)
+      .map((note) => {
+        return <Note key={note.id} note={note} deleteNote={deleteNote} archiveNote={archiveNote} isArchived={true} />;
+      });
+  };
```

Comment:
When defining a component in React, it is common to first define the logic of the component and then just HTML Markup
Here the logic and the view are mixed because a variable is storing a component that will be rendered